packages/client/subscriptions/MeetingSubscription.ts

```diff
@@ -189,21 +190,4 @@ const updateHandlers = {
   SetMeetingMusicSuccess: setMeetingMusicMeetingUpdater
 } as const
 
-const MeetingSubscription = (
-  atmosphere: Atmosphere,
-  variables: MeetingSubscription$variables,
-  router: {history: RouterProps['history']}
-) => {
-  atmosphere.registerSubscription(subscription)
-  return requestSubscription<TMeetingSubscription>(atmosphere, {
-    subscription,
-    variables,
-    updater: subscriptionUpdater('meetingSubscription', updateHandlers, atmosphere),
-    onNext: subscriptionOnNext('meetingSubscription', onNextHandlers, atmosphere, router),
-    onCompleted: () => {
-      atmosphere.unregisterSub(MeetingSubscription.name, variables)
-    }
-  })
-}
-MeetingSubscription.key = 'meeting'
```

Comment:
This was wrong, as it should be `'MeetingSubscription'` here, just as the operation name